.gitignore

```diff
@@ -30,6 +30,14 @@ sync-local.sh
 .exrc
 .helix/
 
+# Android Studio
+.gradle/
+.cxx/
+gradlew
+gradlew.bat
+gradle/wrapper/gradle-wrapper.jar
+local.properties
```

Comment:
Most of these are coming from Gradle, not Android Studio. (Gradle can be used without Android Studio, Android Studio can be used without Gradle.)

`./gradlew`, `gradlew.bat`, and `gradle-wrapper.jar` [are actually supposed to be committed to the repo](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.wrapper.Wrapper.html).

Reply:
Sure but I don't want to commit a .jar to the repo :(

Reply:
I don't like that practice either (same as I honestly don't like many things about Gradle), but that at least warrants a comment.

Reply:
Eh. I added the jar, and a comment about why I'm not including the batch file (we can't even build form cmd.exe)